yoti_python_sdk/doc_scan/exception/doc_scan_exception.py

```diff
@@ -56,5 +63,32 @@ def content(self):
         """
         return self.__response.content
 
+    def __format_response_message(self, response):
```

Comment:
Do you think we should have `if response.headers.get("Content-Type") == "application/json":` outside of this function? And then we can call this `__format_json_response_message` to better reflect what it's doing.

Reply:
I've split this method in e64441c - I think it's a little bit clearer now